lib/browser/api/exports/electron.js

```diff
@@ -1,13 +1,13 @@
-const common = require('../../../common/api/exports/electron')
+const common = require('@electron/internal/common/api/exports/electron')
```

Comment:
This is just a side effect, but I like the absolute pathing this introduces vs the relative pathing we were using.